in_text/text_01_spatial.yml

```diff
@@ -76,130 +82,128 @@ entities:
-      attr-defs: This data release
-      data-min: NA
-      data-max: NA
-      data-units: NA
-    -
-      attr-label: sitetype
-      attr-def: >-
-        Site type from the data source. This includes site_tp_cd from NWIS. Site types include unclassified (blank), springs (SP), streams (ST), tidal streams (ST-TS), canals (ST-CA), ditches (ST-DCH).
-      attr-defs: This data release
-      data-min: NA
-      data-max: NA
-      data-units: NA
-    -
-      attr-label: source
-      attr-def: >-
-        one of four sources from which the temperature data were compiled, including NWIS (NWISdv and NWISuv), WQP, EcoSHEDS, and NorWeST      
-      attr-defs: This data release
-      data-min: NA
-      data-max: NA
-      data-units: NA
-    -
-      attr-label: origsource
-      attr-def: >-
-        WQP, EcoSHEDS, and NorWeST are compliations of temperature data from other sources. This identifies the original collection organization.
-      attr-defs: This data release
-      data-min: NA
-      data-max: NA
-      data-units: NA
-    -
-      attr-label: subsegid
-      attr-def: >-
-        The reach identifier of the matched stream reach. subsegid is an alternative unique reach identifier which was created in order to split segments such that 
-        segment start and endpoints were connected by one reach with no intersecting reaches. The ID was created
-        using the seg_id from the Geospatial Fabric and an index from one to the number of splits (e.g., {subseg_id}_{index}). 
-        This feature corresponds to subseg_id in other files.
-      attr-defs: This data release
-      data-min: NA
-      data-max: NA
-      data-units: NA
-    -
-      attr-label: fishdist
-      attr-def: >-
-        Distance along the stream between the monitoring site and the outlet of the matched stream reach.
-        Positive values indicate the monitoring site is upstream of the outlet of the matched stream reach.
-        Negative values indicate the monitoring site is downstream of the outlet of the matched stream reach.
-      attr-defs: This data release
-      data-min: -13835.68
-      data-max: 19985.95
-      data-units: meters
-    -
-      attr-label: birddist
-      attr-def: >-
-        The Euclidean distance from the monitoring site to the closest point of the matching segment.
-      attr-defs: This data release
-      data-min: 0
-      data-max: 310.2
-      data-units: meters
-    -
-      attr-label: segidnat
-      attr-def: >-
-        The reach identifier of the matched stream reach. segidnat is the national unique stream 
-        reach identifier. This is the ID named seg_id_nat from the Geospatial Fabric for National Hydrologic Modeling.
-        This feature corresponds to "seg_id_nat" in other data files.
-      attr-defs: >-
-        https://www.sciencebase.gov/catalog/item/535eda80e4b08e65d60fc834
-      data-min: NA
-      data-max: NA
-      data-units: NA
+  # -
+  #   data-name: study_monitoring_sites.zip
```

Comment:
Kept this section just commented out. Want to confirm before deleting, @limnoliver  we do not load in the study monitoring sites, right?